src/Form/FormAnalyzer.js

```diff
@@ -383,6 +385,15 @@ class FormAnalyzer {
         this._isCCForm = Boolean(textMatches && deDupedMatches.size > 1);
         return this._isCCForm;
     }
+
+    /**
+     * Checks if the text is a persistent sign-in text, e.g "stay signed in" or "remember me"
+     * @param {string} text
+     * @returns {boolean}
+     */
+    isPersistentSigninText(text) {
+        return /stay.?signed.?in|remember.?me/i.test(text);
```

Comment:
❓ Should we be using `safeRegexTest` here?

Also, how normalised is the `text` value? Do we need to account for varying whitespace?
```suggestion
        return /stay\s+signed\s+in|remember\s+me/i.test(text);
```

Reply:
`safeRegexTest` is definitely the standard here! Thanks, updated.

I am not worried much about the varying whitespaces. Always can update this logic if we see strange examples of sites.